Rebuild-Black-Business/RBB-Website
# Feature/airtable business submit

# Describe your PR

Related to: https://trello.com/c/tsJAF7Lw/124-add-airtable-to-these-forms-checklist


## Pages/Interfaces that will change

/
/about
/ allies

## Steps to test

1. Check all the business related CTAs on the home page and ensure a business airtable form appears.
2. Check the buisness CTAs on the about page and ensure the biz airtable form appears.
3. Check the allies "report or unlist" modal to ensure the contact email appears.
